packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security.cy.ts

```diff
@@ -0,0 +1,90 @@
+describe("TESTS: Configuration => Subsystems => Elytron OIDC client", () => {
+  let wildfly: string;
+  let ssoServer: string;
+
+  before(() => {
+    cy.startWildflyContainerSecured("standalone.xml")
+      .then((result) => {
+        wildfly = result as string;
+        cy.log("wildfly server endpoint: " + wildfly);
+      })
+      .then(() => {
+        cy.startSsoContainer("sso_server").then((result) => {
+          ssoServer = result as string;
+          cy.log("sso server endpoint: " + ssoServer);
+
+          cy.executeInWildflyContainer(
+            `"/subsystem=elytron-oidc-client/provider=keycloak:add(provider-url=${ssoServer}/realms/wildfly-infra)"`
+          );
+
+          cy.executeInWildflyContainer(
+            `"/subsystem=elytron-oidc-client/secure-deployment=wildfly-management:add(provider=keycloak,client-id=wildfly-management,principal-attribute=preferred_username,bearer-only=true,ssl-required=EXTERNAL)"`
+          );
+
+          cy.executeInWildflyContainer(
+            `"/subsystem=elytron-oidc-client/secure-server=wildfly-console:add(provider=keycloak,client-id=wildfly-console,public-client=true)"`
+          );
+
+          cy.executeInWildflyContainer(`"reload"`);
+        });
+      });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  it("Test OICD authentization without RBAC", () => {
+    cy.visit(`?connect=${wildfly}#home`);
+    cy.get("#username").type("userwithoutrole");
+    cy.get("#password").type("password");
+    cy.get("#kc-login").click();
+    cy.url().should(`include`, `localhost:${Cypress.env("HAL_CONTAINER_PORT") as string}`);
+    cy.get(
+      "#hal-header-container > div.collapse.navbar-collapse.hal-header-collapse > ul.nav.navbar-nav.navbar-utility > li:nth-child(5) > a"
+    )
+      .click()
+      .get("#logout-link")
+      .click();
+  });
+
```

Comment:
We should add sanity test with unsuccessful login, like

`it("Fails to log in with bad credentials"), ...`